app/model/Travel/Command.php

```diff
@@ -260,6 +260,20 @@ public function updateTransportTravel(int $id, Money $price, TravelDetails $deta
         $travel->update($price, $details);
     }
 
+    /**
+     * @throws TravelNotFound
+     */
+    public function duplicateTravel(int $id) : void
+    {
+        $travel = $this->getTravel($id);
+
+        if ($travel instanceof VehicleTravel) {
+            $this->addVehicleTravel($travel->getDistance(), $travel->getDetails());
+        } elseif ($travel instanceof TransportTravel) {
+            $this->addTransportTravel($travel->getPrice(), $travel->getDetails());
+        }
```

Comment:
Tady je ještě potřeba pořešit situaci, kdy bysme měli cestu, která je jiného typu (byť ji teď nemáme):
```suggestion
        } else {
            throw new \Model\ShouldNotHappen('Uknown travel type');
        }
```